### ISSUE TITLE
Method `S3::Client::list_objects_v2()` is documented but does not exist

### ISSUE THREAD

**Issue by arimbun** [aws/aws-sdk-ruby#1236]
Error when trying to call the method:

```
undefined method `list_buckets_v2' for #<Aws::S3::Client> (NoMethodError)
```

**Comment by awood45** [comment]
What version of the SDK are you using?

**Comment by arimbun** [comment]
V2

**Comment by awood45** [comment]
What specific gem version are you using? This is a relatively new method, if you're using an older version of the SDK, even within the major version of 2, you may not have this.

**Comment by arimbun** [comment]
Right. I was using v2.2.37. I see that this feature was only released in v2.3.2. I guess you can close this issue.
